dev/io.openliberty.org.apache.myfaces.4.0/src/org/apache/myfaces/config/FacesConfigurator.java

```diff
@@ -147,6 +147,7 @@
 import org.apache.myfaces.spi.ResourceLibraryContractsProviderFactory;
 import org.apache.myfaces.util.ExternalSpecifications;
 import org.apache.myfaces.util.NavigationUtils;
+import org.apache.myfaces.util.Purgeable;
```

Comment:
import java.lang.reflect.InvocationTargetException;
import java.lang.reflect.Method;

Both of these were removed in the upstream changes.

Reply:
This applies across all versions you updated.